media-video/wireplumber/wireplumber-9999.ebuild

```diff
@@ -54,6 +54,23 @@ RDEPEND="${DEPEND}"
 
 DOCS=( {NEWS,README}.rst )
 
+presetfile=30-${PN}.preset
+
+src_prepare() {
+	default
+
+	einfo "Generating ${presetfile}"
+	cat > ${presetfile} <<- EOF || die
```

Comment:
Why are we generating what is essentially a static file at runtime? We could just ship it in FILESDIR instead.

Reply:
At the time this was done it was not clear if and what preset file we'd be installing in the long run. Or, as I mentioned to you on Libera/Matrix, which package would do that (since ultimately the intention is to get pipewire[-sound] operational). This alone means that I did not want to provide a file that then needs to be present in every git clone and every rsync mirror and on every Gentoo user repository when it may go away eventually. Worse yet it may possibly change multiple times over the next few versions, leading to multiple of such small files being there on every Gentoo system and mirror for weeks or months.

Finally the code in question is a common pattern in Gentoo media related packages (the old known example being sys-auth/realtime-base), so if this was somehow against Gentoo rules or guidelines, other packages would need fixing too.

Seeing how this is non-actionable suggestion, I will be marking this conversation as resolved but thank you for making me consider more use of FILESDIR in the future packaging work.